docs/UserGuide.md

```diff
@@ -259,8 +259,12 @@ _Clean up Task parameters, some should be put in the feature, while things like
 ### Task parameters
 * A task is specified using two pieces of information: `TASK_DESCRIPTION` and `DATE TIME`.
 * `TASK_DESCRIPTION` can be any non-empty string made of alphanumeric characters.
-* `DATE TIME` must be of the form d-M-yy HHmm, but the time is optional. <br>
-  e.g. `2-7-22 1345`, `28-10-22` are valid dates.
+* `DATE TIME` must be of the form dd-MM-yy HHmm, but the time is optional. <br>
+  e.g. `2-7-22 1345`, `09-4-22`, `08-06-22 0900`, `7-06-22 2130` and `28-10-22` are all valid `DATE TIME`.
+* If a `TIME` of `2400` if provided, it would simply be translated into `0000` hours of the next day. <br>
+  e.g `2-11-22 2400` will be automatically be inferred as `3-11-22 0000`.
+* If the day portion of the date exceeds the last day of that calendar month, it would default to the last day of the month. <br>
+  e.g `31-4-22` will be automatically converted to `30-4-22` or `30-2-20` will be converted to `29-2-20` since 2020 is a leap year.
 * Although time can be omitted, this will result in the task being created with a default time of `0000` hours.
```

Comment:
This info and below should be in the feature section as they are not constraints.